source/form.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML Academy: Пинк</title>
+  </head>
+  <body class="page__body">
+    <header class="page-header">
+      <a href="index.html" class="page-header__logo">
+        <img
+          src="img/pink-logo.svg"
+          width="74"
+          height="23"
+          alt="Логотип Pink"
+        />
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--active">
+              <a href="index.html"> Главная </a>
+            </li>
+            <li class="main-nav__item">
+              <a href="catalog.html"> Фотографии </a>
+            </li>
+            <li class="main-nav__item">
+              <a href="#"> Конкурс </a>
+            </li>
+            <li class="main-nav__item">
+              <a href="#"> HTML ACADEMY </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+      <div class="contest">
+        <div class="contest__wrapper">
+          <h2 class="contest__title">
+            Конкурс на самое безбашенное путешествие с Пинк
+          </h2>
+        </div>
+      </div>
+    </header>
+
+    <main class="page-main">
+      <h1 class="visually-hidden">Форма заявки на конкурс</h1>
+      <section class="contest">
+        <div class="contest__wrapper">
+          <p class="contest-title">
+            Поделитесь своей историей и получите шанс выиграть ценный приз —
+            1000 миль на вашу бонусную карту!
+          </p>
+
+          <form class="form" action="https://echo.htmlacademy.ru" method="post">
+            <!-- Name -->
+
+            <label for="surname">Фамилия:</label>
+            <input
+              type="text"
+              id="surname"
+              name="surname"
+              placeholder="Укажите фамилию *"
+              required
+            />
+            <label for="name">Имя:</label>
+            <input
+              type="text"
+              id="name"
+              name="name"
+              placeholder="Введите ваше имя *"
+              required
+            />
+            <label for="patronymic">Отчество:</label>
+            <input
+              type="text"
+              id="patronymic"
+              name="patronymic"
+              placeholder="Ну и отчество тоже"
+            />
+
+            <!-- Checkbox -->
+
+            <fieldset>
+              <legend>Ваши безбашенные достижения в путешествии</legend>
+              <ul class="achievement-list">
+                <li class="achievement-item">
```

Comment:
Вот так будет правильно:
```suggestion
                <li class="achievement-list__item">
```
БЭМ элемент блока `achievement-list`.